pkg/codegen/internal/test/testdata/external-resource-schema/python/pulumi_example/foo.py

```diff
@@ -0,0 +1,169 @@
+# coding=utf-8
```

Comment:
As far as I understand, we add `Foo` to test Go aliases. Can we avoid generating it for all other languages, as those seem to be simply noise?

Reply:
We can't avoid it for this test as it generates SDKs for all the languages. Instead of modifying this existing test, I reverted this test change, and added a new Go-only `external-go-import-aliases` test.